docs/source/api/apollo-server.mdx

```diff
@@ -327,7 +294,7 @@ An object containing configuration options for connecting Apollo Server to [Apol
 </td>
 <td>
 
-Controls whether to allow [Batching Queries](../requests/#batching) in a single HTTP Request. Defaults to `true`. If the GraphQL Server has this flag set to `false` and a request comes in formatted as an array rather than as a single request object, an error will be thrown.
+Controls whether to allow [Batching Queries](../workflow/requests/#batching) in a single HTTP Request. Defaults to `false`. If a request comes in formatted as an array rather than as a single request object, an error will be thrown.
```

Comment:
I will the error, great idea. I think I'll just add the message to the end of this sentence though, so folks can search and find their error and read the rest to understand why this is happening:

> Controls whether to allow [Batching Queries](../workflow/requests/#batching) in a single HTTP Request. Defaults to `false`. If a request comes in formatted as an array rather than as a single request object, an error will be thrown ( i.e., `Operation batching disabled.`).

Reply:
Sounds great. I'd also add:
>an error will be thrown ( i.e., `Operation batching disabled.`) _unless batching is enabled_.